Tests/NostrSDKTests/Helpers/JSONEquality.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  JSONEquality.swift
+//  
+//
+//  Created by Joel Klabo on 5/26/23.
+//
+
+import Foundation
+
+func areEqualJSONArrayStrings(_ first: String?, _ second: String?) -> Bool {
```

Comment:
Test coverage needed on these. And they are declared globally. Maybe they could be in a protocol extension?

Reply:
Yeah, much better. Fixed: 5f8947f